.github/workflows/tag-releases-with-prefix.yaml

```diff
@@ -0,0 +1,52 @@
+name: Release
+
+on:
+  workflow_call:
+    inputs:
+      directory:
+        required: true
+        type: string
+      suffix:
+        required: false
+        type: string
+        default: ""
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.matrix.outputs.value }}
+    steps:
+      - uses: actions/checkout@v4
+      - name: Find all module names/locations
+        run: ls ${{ inputs.directory }} | jq -R -s -c 'split("\n")[:-1]' | yq -p=json '.[] as $i ireduce ({}; .[$i] = ["${{ inputs.directory }}" + $i + "${{ inputs.suffix }}"])' > .changed_files.yaml
```

Comment:
I think this needs a comment, I can't understand this line as is

Reply:
Added a clarifying comment, thanks for the callout